homeassistant/util/shutdown.py

```diff
@@ -0,0 +1,25 @@
+"""Shutdown Home Assistant."""
+
+import logging
+import threading
+
+SHUTDOWN_TIMEOUT = 10
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def deadlock_safe_shutdown() -> None:
```

Comment:
Maybe move this to the thread util module?

Reply:
Will do